src/nanoemoji/color_glyph.py

```diff
@@ -278,12 +285,25 @@ def _painted_layers(
             transforms = tuple(
                 affine_between(paths[0], p, FLAGS.reuse_tolerence) for p in paths[1:]
             )
+
+        success = True
         for path, transform in zip(paths[1:], transforms):
             if transform is None:
-                raise ValueError(
-                    f"{debug_hint} grouped {paths[0]} and {path} but no affine_between could be computed"
+                success = False
+                error_msg = (
+                    f"{debug_hint} grouped the following paths but no affine_between "
+                    f"could be computed:\n  {paths[0]}\n  {path}"
                 )
-        yield PaintedLayer(paint, paths[0].d, transforms)
+                if FLAGS.ignore_reuse_error:
+                    logging.warning(error_msg)
+                else:
+                    raise ValueError(error_msg)
+
+        if success:
+            yield PaintedLayer(paint, paths[0].d, transforms)
+        else:
+            for path in paths:
+                yield PaintedLayer(paint, path.d)
```

Comment:
I don't think you want to drop all reuse when a subset fail to compute a transform. Maybe tweak to something roughly like:

```python
# looping twice should be fine, small #s
no_transform = [p for p,t in zip(...) if t is None]
# log or fail as per --ignore_reuse_error; can now report multiple failures in ValueError which is nice
yield PaintedLayer(paint, paths[0].d, [t for t in ... if t is not None])
```

Reply:
In retrospect that changes the layer order, so you actually need to find runs where it worked

Reply:
Leaving that for a followon pr because I want to test to see how widespread the reuse dropping is